src/bio2bel/sources/intact.py

```diff
@@ -0,0 +1,622 @@
+# -*- coding: utf-8 -*-
+
+"""Download and convert IntAct to BEL.
+
+Run with ``python -m bio2bel.sources.intact``
+
+
+`IntAct <https://www.ebi.ac.uk/intact/>`_ is a interaction database with information about interacting proteins, their relation, and the experiments, in which these interactions were found.
+Among the interactions that are documented in IntAct are protein modifications, associations, direct interactions, binding interactions and cleavage reactions.
+These interactions were grouped according to their biological interpretation and mapped to the corresponding BEL relation. The interactions in IntAct had a higher granularity than the interactions in BioGRID. Due to the default BEL namespace of protein modifications :data:`pybel.language.pmod_namespace`, the post-translational protein modification can be identified very accurately. For example, the glycosylation of a protein can be described in BEL by :data:`pybel.dsl.ProteinModification('Glyco').  Although many protein modifications had corresponding terms in BEL, there were some interaction types in IntAct that could not be mapped directly, like 'gtpase reaction' or 'aminoacylation reaction'.
```

Comment:
Sphinx won't execute python code for you, so it's best to write the glycosylation pmod as an inline code block between double backticks

Reply:
you can do
```bash
cd docs
make html
open build/html/index.html
```

to check out how it looks after the docs are built

Reply:
```make html```
does not work because the command is not found

Reply:
I tried to check this, but when I tried to ```make html```, it gives me this:
```python-traceback
Extension error:
Could not import extension sphinx_click.ext (exception: No module named 'sphinx_click')
make: *** [hmtl] Error 2```